charts/scaffold/README.md

```diff
@@ -1,55 +1,45 @@
 # scaffold
 
-![Version: 0.3.21](https://img.shields.io/badge/Version-0.3.21-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
+![Version: 0.4.0](https://img.shields.io/badge/Version-0.4.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
 
-Umbrella chart for deploying the components of the [sigstore](https://www.sigstore.dev/) architecture.
 
-The following components are also included as either direct components or through chart dependencies:
+Scaffolding the components of the sigstore architecture
 
-* [ctlog](https://github.com/google/certificate-transparency-go)
-* [Fulcio](https://docs.sigstore.dev/fulcio/overview/)
-* [MySQL](https://www.mysql.com)
-* [Redis](https://redis.io)
-* [Rekor](https://docs.sigstore.dev/rekor/overview/)
-* [Trillian Log Server](https://github.com/google/trillian)
-* [Trillian Log Signer](https://github.com/google/trillian)
+**Homepage:** <https://sigstore.dev/>
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| The Sigstore Authors |  |  |
 
 ## Requirements
 
 | Repository | Name | Version |
 |------------|------|---------|
 | https://sigstore.github.io/helm-charts | ctlog | 0.2.27 |
-| https://sigstore.github.io/helm-charts | fulcio | 0.4.12 |
+| https://sigstore.github.io/helm-charts | fulcio | 0.5.1 |
 | https://sigstore.github.io/helm-charts | rekor | 0.3.21 |
 | https://sigstore.github.io/helm-charts | trillian | 0.1.10 |
-
-## Quick Installation
-
-```shell
-helm dependency update .
-helm install [RELEASE_NAME] .
-```
-
-This command deploys the default configuration for the scaffold chart. The [Parameters] section describes the various ways in which the chart can be configured.
-
-## Uninstallation
-
-```shell
-helm uninstall [RELEASE_NAME]
-```
-
-The previous command removes the previously installed chart.
+| https://sigstore.github.io/helm-charts | tuf | 0.1.0 |
```

Comment:
Yes, this is what `helm-docs` put out. I didn't re-add the sections because they didn't carry a lot of value imho (installing a chart is a simple command). I brought back these sections in  329d7b4.